src/Bicep.Core/TypeSystem/Azrm/AzrmResourceTypeProvider.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Bicep.Core.Extensions;
+using Bicep.Core.Resources;
+
+namespace Bicep.Core.TypeSystem.Azrm
+{
+    public class AzrmResourceTypeProvider : IResourceTypeProvider
+    {
+        private static IEnumerable<TypeProperty> GetCommonResourceProperties(ResourceTypeReference reference)
```

Comment:
>GetCommonResourceProperties [](start = 49, length = 27)

These aren't all the common properties, though. What about sku and others?

Reply:
Oh never mind, you're still calling the original method to create the fallback type.

---
In reply to: [493901412](https://github.com/Azure/bicep/pull/553#discussion_r493901412) [](ancestors = 493901412)